servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -85,7 +85,8 @@
 
   public TieredBlockStore(TachyonConf tachyonConf) throws IOException {
     mTachyonConf = Preconditions.checkNotNull(tachyonConf);
-    mMetaManager = BlockMetadataManager.newBlockMetadataManager(mTachyonConf);
+    mMetaManager =
+        Preconditions.checkNotNull(BlockMetadataManager.newBlockMetadataManager(mTachyonConf));
```

Comment:
actually, I don't think the return value of `newBlockMetadataManager` here will be NULL anyway. What about checking the NULL metaDataManager in `AllocatorFactory` and `EvictorFactory` ?